Changelog.md

```diff
@@ -36,8 +36,9 @@ and this project follows to [Ragnarök Versioning Convention](https://gist.githu
 
 ### Changed
 
-- Overhauled textures of AAC Honey Badger, Beowulf, HK 417, M16A1, 100 Rounds NATO Drum Magazine, Aimpoint Comp M5 Sight and the pink camo skin
+- Overhauled textures of AAC Honey Badger, Beowulf, HK 417, M16A1, 100 Rounds NATO Drum Magazine and the pink camo skin
 - Overhauled animations of the M40A6
+- Overhauled the texture of the Leupold scope, holographic sight, Aim Point Comp M5 sight, OKP-7 sight, Bijia Reflex sight, RMR sight and Eotech Hybrid sight
```

Comment:
Could you be more precise about the `holographic sight`

Reply:
> Could you be more precise about the `holographic sight`

Once this is done it can be merged, there is some stuff that should be different by since the documentation isn't released I won't make you wait for it.

Reply:
> Could you be more precise about the `holographic sight`

Can't be more specific xDDD